src/nanoarrow/ipc/reader_test.cc

```diff
@@ -115,8 +115,13 @@ TEST(NanoarrowIpcReader, InputStreamFile) {
   ASSERT_EQ(ArrowIpcInputStreamInitFile(&stream, nullptr, 1), EINVAL);
 
   uint8_t input_data[] = {0x01, 0x02, 0x03, 0x04, 0x05};
+#ifdef _WIN32
+  FILE* file_ptr;
+  ASSERT_EQ(tmpfile_s(&file_ptr), 0);
```

Comment:
I'm not sure if Windows expects tmpfile_s to be explicitly closed or not; at least in their documentation that does not seem to be called:

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/tmpfile-s?view=msvc-170